diagnosis/getkuberneteslogs.sh

```diff
@@ -128,56 +121,96 @@ done
 if [ -z "$USER" ]
 then
     echo ""
-    echo "[ERR] --user is required"
+    echo -e "$(date) [Err] --user is required"
     printUsage
 fi
 
 if [ -z "$IDENTITYFILE" ]
 then
     echo ""
-    echo "[ERR] --identity-file is required"
+    echo -e "$(date) [Err] --identity-file is required"
     printUsage
 fi
 
-if [ -z "$DVM_HOST" -a -z "$MASTER_HOST" ]
+if [ -z "$DVM_HOST" ]
 then
     echo ""
-    echo "[ERR] Either --vmd-host or --master-host should be provided"
+    echo -e "$(date) [Err] --vmd-host is required"
     printUsage
 fi
 
+if [ -z "$MASTER_HOST" ]
+then
+    echo ""
+    echo -e "$(date) [Err] --master-host should be provided when available. Functions which require this will be skipped"
+fi
+
 if [ ! -f $IDENTITYFILE ]
```

Comment:
can't we use try to use default id_rsa file is not passed ?.

Reply:
not sure how this would work.